clear-cache.sh

```diff
@@ -18,7 +18,7 @@ install() {
 init() {
     removeFolders
     clearWatchman
-    install
+    #install
```

Comment:
Why not remove install completely?

Reply:
My bad. The comment should not be there. :)